sound/soc/sof/topology.c

```diff
@@ -1073,7 +1073,7 @@ static int sof_control_load_volume(struct snd_soc_component *scomp,
 	scontrol->cmd = SOF_CTRL_CMD_VOLUME;
 
 	/* extract tlv data */
-	if (get_tlv_data(kc->tlv.p, tlv) < 0) {
+	if (!kc->tlv.p || get_tlv_data(kc->tlv.p, tlv) < 0) {
```

Comment:
I am wondering does it really happen?

Reply:
@bardliao yes, if there's no TLV in topology, it will happen. I ran into the problem when testing tplg2.0 without TLV